robosuite_env.py

```diff
@@ -241,10 +279,14 @@ def step(self, action):
         action = self._pre_action(action)
 
         self._apply_action(action)
-        # print(action)
+
+        # TODO: This should not be executed here if we are using RCAN *****
+        # TODO: we should use time.time() from before _pre_action
+
         end_time = time.time() + self.control_timestep
         while time.time() < end_time:
-            NotImplemented
+            time.sleep(0.01)
```

Comment:
Is there a reason we are doing a sleep here? NotImplemented I imagine runs faster

Reply:
I think we need to change this logic anyway cause it is enforcing the control rate at the wrong place. It should take into account the total time applying the control took right?